core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -204,6 +204,10 @@ public Slave getNode() {
         }
     }
 
+    public TaskListener getListener() {
```

Comment:
should really have a mininimal bit of Javadoc as it is a new public method in core.